foundation-models/openai/src/test/resources/chatCompletionResponse.json

```diff
@@ -1,12 +1,13 @@
 {
-  "created": 1719300073,
-  "id": "chatcmpl-9dumHtDEyysGFnknk17n4Lt37tg7T",
-  "model": "gpt-4-32k",
+  "content": "I'm an AI and cannot answer that question as beauty is subjective and varies from person to person.",
```

Comment:
There is a new field `content` which is not in the [Azure spec](https://github.com/Azure/azure-rest-api-specs/blob/30ec6607bbcbeceb13b634de955667c5aa45766a/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-02-01/inference.json#L2456), neither in the [OpenAI documentation](https://platform.openai.com/docs/api-reference/chat/object)

Also, I found no place that documents the changes to `selfHarm`.